README.md

```diff
@@ -148,7 +148,21 @@ Interpretation:
 
 The Steps Stats are based on the cards from the selected deck or collection that were first reviewed in the last 1 month/year or deck life and have at least two reviews.
 
-These stats are helpful for fine-tuning your (re)learning steps to achieve your desired retention in the short-term reviews. To learn more about learning steps, please read [learning steps](https://docs.ankiweb.net/deck-options.html#learning-steps) in the  Anki manual.
+These stats are helpful for fine-tuning your (re)learning steps to achieve your desired retention in the short-term reviews. 
+
+How your learning steps affect the intervals:
+- If you firstly rate again, the interval is the 1st learning step.
```

Comment:
- If your first rating for a new card is Again, the interval is the 1st learning step.
- If your first rating is Again and the second one is Good, the interval is the 2nd learning step.
- If your first rating is Good, the interval is the 2nd learning step.
- If your first rating is Hard, the interval is (1st learning step + 2nd learning step) / 2.